lib/utils/async.js

```diff
@@ -266,6 +268,7 @@ export const nextFrame = (() => {
 /* eslint-disable no-redeclare */
 // rest params can't be placed before non-rest params, so this is unfortunately necessary
 // add another case if you need more params
+declare function frameThrottle<A, B, C>(callback: (a: A, b: B, c: C, start: number) => void): () => void;
```

Comment:
@erikdesjardins I only got this working by putting the case above the others. Is it correct?